### ISSUE TITLE
"Bob" <bob@hi.com> is parsed to "bob@hi.com" <Bob>

### ISSUE THREAD

**Issue by ben-manes** [bbottema/email-rfc2822-validator#12]
```java
var address = EmailAddressParser.getInternetAddress(
    "\"Bob\" <bob@hi.com>", DEFAULT, /* cfws */ true);
System.out.println(l.getAddress());
```
results in `Bob`, not the expected `bob@hi.com`. This means if the personal name is not provided then the email is `null`, which is what caused my code to NPE unexpectedly :)

**Comment by ben-manes** [comment]
This worked in v1.0.1 and v1.1.0, and broke in v1.1.1.

**Comment by chconnor** [comment]
Whoa, weird. @bbottema -- thoughts?

**Comment by bbottema** [comment]
I'll look into it asap.

**Comment by bbottema** [comment]
Hey guys, I'm terribly sorry. I had cleaned up the code a bit for a fix I later reverted, but I left in the cleanup. Somehow I mixed up the argument order of the `InternetAddress` constructor:

```
return new InternetAddress(parts[0], parts[1] + "@" + parts[2]);
// instead of:
return new InternetAddress(parts[1] + "@" + parts[2], parts[0]);
```
I guess I assumed the name came first (maybe because that's how it is in Simple Java Mail as well).

Anyway I fixed it and released 1.1.2.

**Comment by ben-manes** [comment]
Thanks for the speedy fix! Does this prompt a new release of `simple-java-mail` since this is a transitive dependency?

**Comment by bbottema** [comment]
Indeed, I just released 5.1.3! Thanks for the reminder ;)

**Comment by chconnor** [comment]
Awesome, thanks!
